tests/xdsl_opt/test_xdsl_opt.py

```diff
@@ -21,6 +21,7 @@ def test_opt():
         "stencil-to-local-2d-horizontal",
         "frontend-desymrefy",
         "dce",
+        "riscv-regalloc",
```

Comment:
I prefer verbose options, so would suggest something like this:
```suggestion
        "riscv-allocate-registers",
```

Reply:
Ok!